doc/nrf/releases_and_maturity/migration/migration_guide_2.8.rst

```diff
@@ -21,15 +23,81 @@ This document describes the changes required or recommended when migrating your
+
+Build and configuration system
+==============================
+
+.. toggle::
+
+   * Sysbuild now handles the following MCUboot image ID assignments:
+
+     * MCUboot updates (using b0) are automatically assigned to MCUboot.
+	    The ``SB_CONFIG_MCUBOOT_UPDATEABLE_IMAGES`` Kconfig option must not be incremented to include this image.
+     * Applications and MCUboot must use the MCUboot assigned image ID Kconfig values to refer to image IDs instead of hardcoding them.
+     * Depending upon enabled images, some image IDs might differ in |NCS| 2.8 and higher than from previous releases.
+
+
+Nordic Secure Immutable Bootloader (NSIB, B0, or B0n)
+=====================================================
+
+.. toggle::
+
+   * Custom printing has been dropped in favor of using the logging subsystem, with output printed out to the default logging device.
+     The ``CONFIG_SECURE_BOOT_DEBUG`` Kconfig option has been removed.
+     To disable logging in B0 or B0n, set the :kconfig:option:`CONFIG_LOG` option to ``n``.
+     To send logs over RTT instead of UART, apply the following settings:
+
+     * Enable the :kconfig:option:`CONFIG_USE_SEGGER_RTT` and :kconfig:option:`CONFIG_RTT_CONSOLE` Kconfig options.
+     * Disable the :kconfig:option:`CONFIG_UART_CONSOLE` and :kconfig:option:`CONFIG_SERIAL` Kconfig options.
+
+nRF70 Series
+============
+
+.. toggle::
+
+   * The nRF70 Series support is now part of Zephyr upstream and it requires the following changes:
+
+     * The nRF70 Series driver namespace has been renamed from ``NRF700X`` to ``NRF70``.
+       For example, ``CONFIG_NRF700X_RAW_DATA_RX`` to ``CONIFG_NRF70_RAW_DATA_RX``.
+       Update your application configurations to use the new namespace.
+
+     * The nRF70 Series driver now uses per-module kernel heap with a higher default.
+       If a sample or an application uses the kernel heap but uses less than the default size, a build warning is displayed.
+       Use the :kconfig:option:`CONFIG_HEAP_MEM_POOL_IGNORE_MIN` Kconfig option and enable it to suppress the warning.
+
+   * The WPA supplicant is also now part of Zephyr upstream and it requires the following changes:
+
+     * The WPA supplicant namespace has been renamed from ``WPA_SUPP`` to ``WIFI_NM_WPA_SUPPLICANT``.
+       For example, ``CONFIG_WPA_SUPP=y`` to ``CONFIG_WIFI_NM_WPA_SUPPLICANT=y``.
+       Update your application configurations to use the new namespace.
+
+   * The SR co-existence feature should now be explicitly enabled using the :kconfig:option:`CONFIG_NRF70_SR_COEX` Kconfig option.
+     The RF switch feature should be enabled using the :kconfig:option:`CONFIG_NRF70_SR_COEX_RF_SWITCH` Kconfig option.
+
+nRF54L Series
+=============
+
+.. toggle::
+
+   * Use the :ref:`ZMS (Zephyr Memory Storage) <zephyr:zms_api>` storage system for all devices with RRAM memory technology, such as the nRF54L Series devices.
```

Comment:
bit odd that this mentions for all nrf54l devices when it's under the nrf54l series heading

Reply:
To add to this: why `devices` plural? There's only one so far: nRF54L15, with some revisions.

Reply:
considering the nRF54L05 and nRF54L10. But I have removed this part now.